specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2023-09-01/logicalNetworks.json

```diff
@@ -0,0 +1,655 @@
+    },
+    "IPPoolInfo": {
+      "type": "object",
+      "properties": {
+        "used": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Number of IP addresses allocated from the IP Pool"
+        },
+        "available": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Number of IP addresses available in the IP Pool"
+        }
+      }
+    },
+    "IPPool": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the IP-Pool"
+        },
+        "ipPoolType": {
+          "type": "string",
+          "enum": [
+            "vm",
+            "vippool"
+          ],
+          "x-ms-enum": {
+            "name": "IPPoolTypeEnum",
+            "modelAsString": false
+          },
+          "description": "Type of the IP Pool [vm, vippool]"
+        },
+        "start": {
+          "type": "string",
+          "description": "Start of the IP address pool"
+        },
+        "end": {
+          "type": "string",
+          "description": "End of the IP address pool"
+        },
+        "info": {
+          "$ref": "#/definitions/IPPoolInfo"
+        }
+      }
+    },
+    "RoutePropertiesFormat": {
+      "type": "object",
+      "properties": {
+        "addressPrefix": {
+          "type": "string",
+          "description": "The destination CIDR to which the route applies."
+        },
+        "nextHopIpAddress": {
+          "type": "string",
+          "description": "The IP address packets should be forwarded to. Next hop values are only allowed in routes where the next hop type is VirtualAppliance."
+        }
+      },
+      "description": "RoutePropertiesFormat - Route resource."
+    },
+    "Route": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RoutePropertiesFormat",
+          "description": "Properties of the route."
+        },
+        "name": {
+          "type": "string",
+          "pattern": "^[a-zA-Z0-9]$|^[a-zA-Z0-9][-._a-zA-Z0-9]{0,78}[_a-zA-Z0-9]$",
+          "description": "Name - name of the subnet"
+        }
+      },
+      "description": "Route - Route resource."
+    },
+    "RouteTablePropertiesFormat": {
+      "type": "object",
+      "properties": {
+        "routes": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Route"
+          },
+          "description": "Collection of routes contained within a route table.",
+          "x-ms-identifiers": []
+        }
+      },
+      "description": "RouteTablePropertiesFormat - Route Table resource."
+    },
+    "RouteTable": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Etag - Gets a unique read-only string that changes whenever the resource is updated."
```

Comment:
why is the etag called as ID ? Is there a mismatch between the name and description ?

Reply:
Yep ideally should be named etag. I will mark it as a won'tfix for this api-version however since the property description is still accurate enough to avoid user misunderstanding but is a property name change that will trigger bottom stack changes. Let me know if you think otherwise. 

#won'tfix

Reply:
So this being a stable API version, you're not going to be able to rename it in the future without it being a breaking change. You should probably fix it now or never. @pathuang-sia.

Reply:
What I guess you *could* do, in lieu of a breaking change, is add an additional etag property and mark this old property deprecated, to make your API easier to undersand in the future, without breaking existing customers. But why not just avoid such a mess altogether?

Reply:
Yeah it makes more sense to do it right now. I spoke with the scenario owner and we will fix the property name in this api-version.

Reply:
fixed